lib/user.rb

```diff
@@ -63,17 +57,6 @@ def next_command(previous)
     end
   end
 
-  def validate_input(p_input)
-    # note to self: imlpement map to replace this
-    p_input.each do |value|
-      if /[1-6]/ === value.to_s
-        # do nothing
-      else
-        return false
-      end
-    end
-  end
-
   def is_quit?(user_input)
     if (user_input.downcase.eql? 'quit')
       # quit_plz
```

Comment:
So it looks like `User` is still doing way too much. Remember to ask yourself the question "What is the responsibility of this class?" and answer in a simple sentence. If you use the word "and" anywhere in that sentence, that should set off a red flag.

Reply:
Also, where are your tests for `User`? ❓